turbinia/workers/artifact.py

```diff
@@ -26,7 +26,7 @@
 class FileArtifactExtractionTask(TurbiniaTask):
   """Task to run image_export (log2timeline)."""
 
-  def __init__(self, artifact_name):
+  def __init__(self, artifact_name='FileArtifact'):
```

Comment:
Just curious, why is this getting set?  Is this the default name used by most Jobs that create this Task?

Reply:
When we use `getattr` to deserialize an object, we don't know ahead of time what its arguments were set to. This lets us instantiate the object with some default, which we can then override when updating the object's `__dict__'.

Reply:
Ah, ok.  Thanks.